ios/Surveys/Sources/Presentation/Modules/SurveyQuestions/Views/DropdownAnswerView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DropdownAnswerView.swift
+//  Surveys
+//
+//  Created by Su Ho on 29/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+// MARK: - DropdownAnswerView
+
+struct DropdownAnswerView: View {
+
+    @State private var selectedInput: String
+
+    private let ids: [String]
+    private let answers: [Answer]
+    private var input: AnswerInput
+    private let inputDidChange: (AnswerInput) -> Void
+
+    var body: some View {
+        Picker(String(describing: Self.self), selection: $selectedInput) {
+            ForEach(ids, id: \.self) { id in
+                let font = selectedInput == id
+                    ? Typography.neuzeitSLTStdBookHeavy.font(size: 20.0)
+                    : Typography.neuzeitSLTStdBook.font(size: 20.0)
+                Text(answers.first(where: { $0.id == id })?.content ?? "")
+                    .font(font)
+                    .foregroundColor(Color.white)
+            }
+        }
+        .pickerStyle(.wheel)
+        .padding(20.0)
```

Comment:
Just a concern relating to the space between the 2 items. Could we increase it to match the design?

Reply:
We can't. We are using the default picker, it's not supported for fully customization 😢